### ISSUE TITLE
interactive 'mix colors' option in bottom pane

### ISSUE THREAD

**Issue by thezbyg** [thezbyg/gpick#44]
Original [issue 44](https://code.google.com/p/gpick/issues/detail?id=44) created by thezbyg on 2011-07-10T08:53:02.000Z:

While often I want to make blends in the palette, it's very common that I end up going back and forth tuning the blend endpoints. For this, the 'mix colors' dialog is troublesome, because I have to repeatedly invoke it every time I change the endpoint colors.

If instead I could interact with the endpoints to directly tune them, and finally commit the result to the palette, that would be a lot more useful IMO, and would also make paint-program users,
of which I am one, happy (providing a 'blender' that can easily be dropped to or picked from.) 

tl;dr version : An interactive version of 'mix colors' as a choice in the bottom pane would be very useful, IMO.
